pkgs/development/ocaml-modules/camlimages/default.nix

```diff
@@ -17,8 +15,10 @@ buildDunePackage rec {
     sha256 = "1m2c76ghisg73dikz2ifdkrbkgiwa0hcmp21f2fm2rkbf02rq3f4";
   };
 
+  duneVersion = "3";
+
   nativeBuildInputs = [ cppo ];
-  buildInputs = [ dune-configurator graphics lablgtk stdio ];
+  buildInputs = [ dune-configurator graphics lablgtk stdio findlib ];
```

Comment:
This change isn't related to this PR because it fixes master bug.

Reply:
Might as well fix it while we're in here